src/int.cr

```diff
@@ -117,6 +117,7 @@ struct Int
     self.class.new(to_f // other)
   end
 
+  @[Deprecated("Int#/ will return a Float. Might need to use Int#// for integer division.")]
```

Comment:
I'd prefer to mention when this is expected to change.
Im not sure if we settled on that, though. 0.29.0 or give it more time?

Reply:
As soon as the compiler is ready. I believe 0.28 is ready, and announces the deprecation, so 0.29 will introduce the breaking change.

Reply:
We can say it's schedule in 0.29, but I haven't prepared the change to fully check if the compiler is ready.

And if some place of the std a replacement is missing then we need a patch or minor release in the middle probably.

So either 0.29.0 with the risk of forcing 0.28.1 or delaying until 0.30.0.
Or 0.30.0 and might need to delay more than necessary.

Both (or none) works for me.

Reply:
Let's just say 0.29.0. If it should be further delayed for some reason, it's not a big issue.

Reply:
Done